fp.js

```diff
@@ -18,11 +19,13 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person) {
-  if(person.age < 35){
-    peopleYoungerThan35.push(person)
-  }
-});
+// people.forEach(function(person) {
+//   if(person.age < 35){
+//     peopleYoungerThan35.push(person)
+//   }
+// });
+  peopleYoungerThan35 = people.filter((x,i)=>people[i].age<35)
+
```

Comment:
Thanks James, I'm still trying to figure out ES6.  In this case I used non-semantic because I was getting through it, but I will make sure to be more semantic in the future.